### PR TITLE
Alternative to Cloudinary blog

### PR DESCRIPTION
**Pull request by SnezhannaM**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/appwrite-vs-cloudinary/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Appwrite Storage: open-source alternative to Cloudinary"
+description: Get a comprehensive comparison between Appwrite and Cloudinary to decide which platform fits your media and files needs best.
+date: 2024-09-03
+cover: /images/blog/appwrite-compared-to-supabase/cover.png
```
Comment (adityaoberai) [inline src/routes/blog/post/appwrite-vs-cloudinary/+page.markdoc:6]: @SnezhannaM is this correct?

Comment (SnezhannaM) [reply src/routes/blog/post/appwrite-vs-cloudinary/+page.markdoc:6]: @adityaoberai yes, all competitor blogs have the same cover

Comment (LauraDuRy) [reply src/routes/blog/post/appwrite-vs-cloudinary/+page.markdoc:6]: @SnezhannaM should we update the name then? it feels a bit weird to have this naming. @adityaoberai thoughts?

Comment (SnezhannaM) [reply src/routes/blog/post/appwrite-vs-cloudinary/+page.markdoc]: @adityaoberai I changed the name and updated this blog and the Supabase one.

**Review by ernstmul — APPROVED** [review]

**Pull request merged**